src/System.Memory/ref/System.Memory.cs

```diff
@@ -245,7 +278,7 @@ public static partial class SequenceReaderExtensions
         public override bool Equals(object obj) { throw null; }
         public override int GetHashCode() { throw null; }
         public static bool operator ==(System.Buffers.StandardFormat left, System.Buffers.StandardFormat right) { throw null; }
-        public static implicit operator System.Buffers.StandardFormat (char symbol) { throw null; }
+        public static implicit operator System.Buffers.StandardFormat(char symbol) { throw null; }
```

Comment:
Was this change made automatically, or accidentally?

Reply:
I ran `dotnet msbuild /t:GenerateReferenceSource` and then fixed up anything where it was over-eager (eg those `unmanaged` issues). This whitespace change looked harmless so I left it.

Reply:
Let me know if I should revert it

Reply:
I tried it myself and it doesn't look like the tool removes the whitespace here. I would revert it.

Reply:
OK, it must have been VS auto-format then, when I manually fixed-up the `unmanaged` issues, 
I will revert in VS Code